### ISSUE TITLE
Таки броски предметов в решетку

### ISSUE THREAD

**Issue by ucnaHez** [animusdev/green#304]
Я все-таки считаю, что они убирают всю идею решетки под током. Ковырять решетку под током стеклышком - еще ничего, бросать в нее кого-нибудь - тоже сложно, и встретит сопротивление кого-нибудь, но вот "иду, вижу - решетка в 80 мегавольт, ну я бросил в нее огнетушитель три раза - он и сломалась" - фу. 

Какой в них тогда вообще смысл? Она не может удержать НИКОГО, чтобы ее сломать - не нужны перчатки, кусачки, полчаса и стекло, просто берешь любой предмет и кидаешь. Кидаешь раз, кидаешь два, кидаешь три - сломано. Даже стекло задержит на больший срок.

**Comment by Euronem** [comment]
ультимативное кря

**Comment by jammer312** [comment]
Ну, как бы, у нас и толстенная броня скоро будет бумажным шприцом пробиваться по твоей инициативе, что уж тут о решеточках говорить.

**Comment by Seldorne** [comment]
Ну, почему бы и не добавить где-нибудь 50% резиста решёточкам от летящих предметов?

**Comment by jammer312** [comment]
Ну а почему бы не сделать то же самое для шприцей?

**Comment by ucnaHez** [comment]
Я не понимаю связи шприцов и решеточек. Это иссуй про решеточки, шприцы в пулреквесте.

**Comment by Seldorne** [comment]
@ucnaHez Нет, ну а что ещё? Только делать их прочнее. Вайтовский вариант, когда надо кидать в решётку с растояния тайла в четыре, чтобы тебя не поразил молнией сам Тор, довольно глупый.
На мой взгляд, решётки вообще больше для защиты от каких-нибудь ксенов.
